soc/arm/nuvoton_npcx/npcx7/Kconfig.defconfig.series

```diff
@@ -35,6 +35,12 @@ config GPIO_NPCX
 	help
 	  Enable support for NPCX GPIO driver.
 
+config PWM_NPCX
```

Comment:
How come this option is defined in two places?

Reply:
The first definition in Kconfig.npcx is introducing this variable. And the second definition makes this variable turn on as default if chip series is npcx 7. You can refer the similar declarations in https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/pwm/Kconfig.xec#L6 and https://github.com/zephyrproject-rtos/zephyr/blob/master/soc/arm/microchip_mec/mec1501/Kconfig.defconfig.mec1501hsz#L46.